source/_integrations/qnap.markdown

```diff
@@ -3,124 +3,31 @@ title: QNAP
 description: Instructions on how to integrate the QNAP sensor within Home Assistant.
 ha_category:
   - System Monitor
-ha_release: 0.38
+ha_release: 2022.10
```

Comment:
The original release of this integration doesn't change by adding a config flow. Please revert this change.